src/test/mimc_test.cpp

```diff
@@ -13,7 +15,147 @@ typedef libff::Fr<ppT> FieldT;
 namespace
 {
 
-TEST(TestRound, TestTrue)
+// Testing that (15212  + 98645 + 216319)**7 =
```

Comment:
Why is this MiMC file showing up here?

Reply:
Any news on that @rrtoledo?

Reply:
I merged round, mimc_mp_test and mimc_test into one file (mimc_test) to clean up as we have more and more tests.

Reply:
Please do this in a separate PR as this is not related to blake2s

Reply:
Related issue opened https://github.com/clearmatics/zeth/issues/84, will be fixed in a sperate PR